renderer/src/components/GoogleConnection/GoogleConnection.tsx

```diff
@@ -156,20 +156,14 @@ const GoogleConnection = ({ isOnline }) => {
         <span className="font-medium dark:text-dark-heading">Google</span>
         {!loggedUsers.length && (
           <Tooltip tooltipText={CONNECTION_MESSAGE} disabled={isOnline}>
-            <Button text="Add account" callback={signInHandler} type="button" disabled={!isOnline}/>
+            <Button
+              text={!loggedUsers.length ? "Add account" : "Add another account"}
+              callback={signInHandler}
+              type="button"
+              disabled={!isOnline}
+            />
```

Comment:
After this change I don't see button "Add another account"
![image](https://github.com/ukad-group/timetracker-desktop-client/assets/110410115/5a9ab5d2-c5f8-42a9-9231-a44f8ad13384)